adapters/libev.h

```diff
@@ -136,8 +164,13 @@ static int redisLibevAttach(EV_P_ redisAsyncContext *ac) {
     ac->ev.addWrite = redisLibevAddWrite;
     ac->ev.delWrite = redisLibevDelWrite;
     ac->ev.cleanup = redisLibevCleanup;
+    ac->ev.scheduleTimer = redisLibevScheduleTimer;
     ac->ev.data = e;
 
+    if (ac->c.timeout->tv_sec != 0 || ac->c.timeout->tv_usec != 0) {
+        redisLibevScheduleTimer(e, *ac->c.timeout);
+    }
+
```

Comment:
As discussed, please consider having this aligned with the libevent behavior (or vice versa if you find the current libevent behavior a bug).

Reply:
Removed these lines to align with our libevent adapter.